packages/frontend/electron/src/main/updater/index.ts

```diff
@@ -10,6 +18,27 @@ export const updaterHandlers = {
   quitAndInstall: async () => {
     return quitAndInstall();
   },
+  downloadUpdate: async () => {
+    return downloadUpdate();
+  },
+  autoCheckUpdate: async () => {
+    return getAllowAutoCheckUpdate();
+  },
+  autoDownloadUpdate: async () => {
+    return getAllowAutoDownloadUpdate();
+  },
+  enableAutoCheckUpdate: async () => {
+    return setAllowAutoCheckUpdate(true);
+  },
+  disableAutoCheckUpdate: async () => {
+    return setAllowAutoCheckUpdate(false);
+  },
+  enableAutoDownloadUpdate: async () => {
+    return setAllowAutoDownloadUpdate(true);
+  },
+  disableAutoDownloadUpdate: async () => {
+    return setAllowAutoDownloadUpdate(false);
+  },
```

Comment:
Please reduce the number of handlers required. Eg., expose a single `setConfig` handler to configure all of these options and a single `getConfig` function to return the latest config.